spec/models/zone_spec.rb

```diff
@@ -192,6 +192,28 @@
     end
   end
 
+  context "maintenance zone" do
+    it "creates missing region during seed" do
+      expect(MiqRegion.my_region).to be_nil
+      described_class.seed
+      expect(MiqRegion.my_region).to be_present
+    end
+
+    it "is seeded with relation to region" do
+      described_class.seed
+      zone = Zone.where("name LIKE ?", "#{Zone::MAINTENANCE_ZONE_NAME_PREFIX}%").first
+
+      expect(described_class.maintenance_zone).to be_present
+      expect(described_class.maintenance_zone.id).to eq(zone.id)
+      expect(MiqRegion.my_region&.maintenance_zone).to eq(zone)
+    end
+
+    it "is not visible" do
+      described_class.seed
+      expect(described_class.maintenance_zone.visible).to be_falsey
```

Comment:
If we're testing `default_value_for` we should make sure that it is `false` and not `nil`

Reply:
ok, my fault, I expected that `be_false` (which is alias) means "be false" - and it doesn't